### ISSUE TITLE
Decide if we will test single-key navigation in VoiceOver's Quick Nav mode

### ISSUE THREAD

**Issue by mfairchild365** [w3c/aria-at#63]
This came up on the call today, and I don't want to lose track of the conversation.

Question: Should we test the single-key navigation in VoiceOver's Quick Nav Mode?

Notes:

- I checked our tests that are in the prototype, and we do not currently test this.
- VoiceOver does not have single-key navigation turned on by default. It is a setting that users can enable. To test:
  1. Reset to default settings:
     1. Open VoiceOver Utiltity
     2. File > Reset All VoiceOver Preferences
  2. Observe the setting that is off by default
     1. Using VoiceOver Utility
     2. Open the "commanders" panel
     3. Click the "quick nav" tab
     4. Observe that the setting "Enable single-key webpage navigation when using Quick Nav" is unchecked

My take: Since this functionality is not enabled by default, we should not expend time testing it.

**Comment by zcorpan** [comment]
@cookiecrook, do you have input on this?
